challenges/easy/who-likes-it/solutions/aldotele/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+func main() {
+  fmt.Println("Enter people below (blank to quit)")
```

Comment:
can you please write an example of how to use this program? Thanks!

Reply:
I changed approach to make it easier and self-explanatory, hope it is more clear now!